client/state/data-layer/wpcom/sites/atomic/software/index.js

```diff
@@ -36,9 +34,6 @@ const receiveInstallError = ( action, error ) => [
 		context: 'atomic_software_install',
 		error: error.error,
 	} ),
-	errorNotice(
-		translate( "Sorry, we've hit a snag. Please contact support so we can help you out." )
-	),
```

Comment:
👍 I think rrror notice should be addressed at the components level.

Reply:
I think the same 👍🏼